components/style/properties/properties.mako.rs

```diff
@@ -2665,6 +2663,14 @@ pub fn apply_declarations<'a, F, I>(device: &Device,
                                                  error_reporter);
             % endif
             }
+
+            let writing_mode = get_writing_mode(context.style.get_inheritedbox());
+            context.style.writing_mode = writing_mode;
```

Comment:
Yeah, but note that that writing mode is not the writing mode property itself. Anyway you're right, and I should probably leave it where it was..